tests/.snapshots/success.txt

```diff
@@ -1067,9 +1083,22 @@
   ✓ nested → it may have static note and runtime note
   // This is before each static note
   // This is describe static note
+  // This is before each describe static note
   // This is a static note within describe
   // This is before each runtime note
+  // This is before each describe runtime note
   // This is a runtime note within describe
+  ✓ nested → describe nested within describe → it may have a static note and runtime note
```

Comment:
Can you add some tests too in the middle of the test file?

Reply:
There is a test after this one: `multiple notes`